src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-SPECVIEW-AMD.json

```diff
@@ -0,0 +1,52 @@
+{
+  "Description": "SPEC view GPU Performance Workload",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "01:00:00",
+    "SupportedPlatforms": "win-x64",
+    "SupportedOperatingSystems": "Windows"
+  },
+  "Parameters": {
+    "GpuModel": "v620"
+  },
+  "Actions": [
+    {
+      "Type": "SpecViewExecutor",
+      "Parameters": {
+        "Scenario": "SPECTestingAllViewSets",
+        "CommandArguments": "-viewset \"3dsmax,catia,creo,energy,maya,medical,snx,sw\" -nogui",
+        "PackageName": "specviewperf2020",
+        "Tags": "GPU,SPECview,Gaming"
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "DependencyPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallAMDGPUDriverPackage",
+        "BlobContainer": "packages",
+        "BlobName": "AMDdependencyv620.zip",
```

Comment:
Thank you for following our pattern. We try to use lower case blobname also, which is amdgpudriver.zip. Unless you have special driver requirement, let me know